packages/react-composites/src/composites/CallComposite/hooks/useAdaptedSelector.ts

```diff
@@ -80,7 +82,9 @@ const memoizeState = memoizeOne(
     deviceManager: DeviceManagerState,
     calls: { [key: string]: CallState },
     latestErrors: CallErrors,
-    displayName?: string
+    displayName?: string,
+    alternateCallerId?: string,
+    environmentInfo?: undefined | /* @conditional-compile-remove(unsupported-browser) */ EnvironmentInfo
```

Comment:
Is it possible to conditional compile the param instead of the type?

Reply:
not without defining a whole type for the function params unfortunately. it wouldn't need it at all if it weren't the EnvironmentInfo type since its not in the stable SDK yet